src/Controllers/GagePageController.ts

```diff
@@ -248,8 +248,20 @@ module StreamStats.Controllers {
         public formattedDailyPlusAvg = [];
         public formattedDischargePeakDates = []; // Stage vs. Discharge Plot
         public dailyValuesOnly = [];
-        public ageQualityData = 'age'; //Stage vs. Discharge Plot
-        public error: any; //Stage vs. Discharge Plot
+        public ageQualityData = 'age';
+        public error: any;     
+        public monthSliderOptions: any;
+        public startMonth: number;
+        public endMonth: number;
+        public startYear: number;
+        public endYear: number;
+        public yearSliderOptions: any;
+        public showQuality: any;
```

Comment:
I don't think this variable is used.

Reply:
Sorry not sure why it highlighted them all, I just mean showQuality doesn't seem used. 
![image](https://github.com/USGS-WiM/StreamStats/assets/59063181/98fb4005-e3e8-481d-8126-b85d5eefe7de)